examples/examples_py_test.go

```diff
@@ -17,31 +17,30 @@
 package examples
 
 import (
-	"fmt"
 	"os"
 	"path"
 	"testing"
 
 	"github.com/pulumi/pulumi/pkg/v3/testing/integration"
 )
 
-func TestAwsPy(t *testing.T) {
```

Comment:
Consider just adding a https://pkg.go.dev/testing#T.Skip `t.Skip("<link to issue>")` instead?